### ISSUE TITLE
Sessions API: Support creating Protocol Engine commands in a `basic` Session

### ISSUE THREAD

**Issue by mcous** [Opentrons/opentrons#8070]
## Overview

As an HTTP API user, I'd like to to issue protocol commands one-at-a-time to my session so that I can run protocol logic (e.g. interfacing with external hardware) on a machine other than the OT-2's onboard computer.

This ticket refers to the experimental `/sessions` routes that are currently (as of the 4.4 release) behind the `enableProtocolEngine` feature flag.

## Acceptance criteria

- [x] The `/sessions` API has a `basic` type session that is created by default
- [x] ~When created, the session is loaded with attached pipettes loaded (blocked by #8069)~ (TODO: write a separate ticket for this. Not needed for MVP because client can issue `loadPipette` command themselves)
- [ ] A client may `POST` to `/sessions/:id/commands` to queue up a command in the Protocol Engine
- [ ] A client may use the existing `POST /sesions/:id/actions` endpoints to start, pause, and resume executing the queued commands

## Implementation details

- Create command endpoint should straightforward: take the CommandRequest and call `engine_store.engine.add_command` (or something like that)
- Currently the `ProtocolRunner` + cleanup wil interfere with keeping the engine active after a play, because it will shut down the engine as soon as the queue is exhausted
    - For a basic session, we want the engine to remain active, even if the queue is empty
    - This will require investigation, start at the backgrounding of `runner.join` in the create session endpoint
